streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamsMetricsImpl.java

```diff
@@ -117,7 +118,7 @@ public Sensor addLatencyAndThroughputSensor(String scopeName, String entityName,
 
         // first add the global operation metrics if not yet, with the global tags only
         Sensor parent = metrics.sensor(sensorName(operationName, null), recordingLevel);
-        addLatencyMetrics(scopeName, parent, "all", operationName, tagMap);
+        addLatencyMetrics(scopeName, parent, null, operationName, tagMap("all", "all"));
```

Comment:
Hmm.. This will result in the mbean object name as, for example:

```
kafka.streams:type=stream-task-metrics,client-id=streams-wordcount-e600ab70-9a78-4886-b2c7-bce8f9fdbcf2-StreamThread-1,all=all
```

and

```
kafka.streams:type=stream-task-metrics,client-id=streams-wordcount-e600ab70-9a78-4886-b2c7-bce8f9fdbcf2-StreamThread-1,streams-task-id=1_0
```

I think it is better to be "streams-task-id=all".